src/skin.rs

```diff
@@ -98,32 +98,50 @@ impl MinecraftSkin {
                     BodyPart::ArmLeft => {
                         match self.version() {
                             MinecraftSkinVersion::Modern => self.0.crop_imm(52, 52, arm_width, 12),
-                            _                            => self.get_part(Bottom, ArmRight)
+                            _                            => self.get_part(Bottom, ArmLeft)
                         }
                     },
                     BodyPart::ArmRight => {
                         match self.version() {
                             MinecraftSkinVersion::Modern => self.0.crop_imm(44, 36, arm_width, 12),
-                            _                            => self.get_part(Bottom, ArmRight),
+                            _                            => self.get_part(Bottom, ArmLeft).fliph(),
                         }
                     },
                     BodyPart::LegLeft => {
                         match self.version() {
                             MinecraftSkinVersion::Modern => self.0.crop_imm(4, 52, 4, 12),
-                            _                            => self.get_part(Bottom, LegRight),
+                            _                            => self.get_part(Bottom, LegLeft),
                         }
                     },
                     BodyPart::LegRight => {
                         match self.version() {
                             MinecraftSkinVersion::Modern => self.0.crop_imm(4, 36, 4, 12),
-                            _                            => self.get_part(Bottom, LegRight),
+                            _                            => self.get_part(Bottom, LegLeft).fliph(),
                         }
                     },
                 }
             },
         }
     }
 
+    pub(crate) fn render_body(&self, overlay: bool) -> DynamicImage {
+        let layer_type = match overlay {
+            true  => Layer::Both,
+            false => Layer::Bottom
+        };
+
+        let mut image = RgbaImage::new( 16, 32);
```

Comment:
nit: remove the space before `16`

Reply:
oop, yeah, of course, I guess I missed that in my proofreading, fixing now...